src/courseware/course/sidebar/sidebars/course-outline/components/CompletionIcon.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+
+import CompletionIcon from './CompletionIcon';
+
+describe('CompletionIcon', () => {
+  test('renders check circle icon when completion is equal to total', () => {
```

Comment:
```suggestion
  it('renders check circle icon when completion is equal to total', () => {
```
`it` is generally used everywhere instead of `test`

Reply:
Changed